addons/website/tests/test_fuzzy.py

```diff
@@ -225,7 +225,7 @@ def test_06_case_insensitive_results(self):
 
     def test_07_no_fuzzy_for_mostly_number(self):
         """ Ensures exact match is used when search contains mostly numbers. """
-        self._create_page('Product P7935432254U7 page', 'Product P7935432254U7', '/numberpage')
+        self._create_page('Product P7935432254U7 page', 'Product P7935432254U7 kangaroo shoes', '/numberpage')
```

Comment:
Moving this commit as first commit otherwise the PR is broken for a few commits if I understand the commit message right